conf/messages.en

```diff
@@ -239,7 +239,7 @@ departure.status.departureDeclarationRejected = Departure declaration rejected
 departure.status.departureDeclarationReceived = Departure declaration received
 departure.status.guaranteeValidationFail = Guarantee rejected
 departure.status.transitDeclarationSent = Transit declaration sent
-departure.status.writeOffNotification = Guarantee written off
+departure.status.writeOffNotification = Movement ended
```

Comment:
Is this P4 or P5?

Reply:
This is P5, this comes from Ken's content ticket: https://github.com/hmrc/manage-transit-movements-frontend/pull/406

Reply:
P4 ones have movement at the start IIRC